common/changes/@stonecrop/beam/feat-beam-toast-notify_2024-10-28-18-12.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@stonecrop/beam",
+      "comment": "added toast notifications",
+      "type": "none"
```

Comment:
@crabinak This isn't documented anywhere, but I think you'll have to manually change this to `"type": "patch"` for generating changelogs